lib/rules/lowercase_description.js

```diff
@@ -0,0 +1,94 @@
+const { isEmpty: _isEmpty, has: _has, keys: _keys } = require("lodash");
+const { format } = require("util");
+const Rule = require("./Rule");
+
+module.exports = class LowercaseDescription extends Rule {
+    static meta = {
+        ruleId: "lowercase_description",
+        message: "Expected lowercase description on keyword '%s'",
```

Comment:
maybe this is enough :thinking: 
```suggestion
        message: "Use lowercase description",
```

Reply:
So should i write `Use lowercase description` or `Use lowercase title`

Reply:
okay